e2e/upgrade-cephfs.go

```diff
@@ -61,7 +61,27 @@ var _ = Describe("CephFS Upgrade Testing", func() {
 			Skip("Skipping CephFS Upgrade Test")
 		}
 		c = f.ClientSet
-		if cephCSINamespace != defaultNs {
+		cephFSDeployment = &CephFSDeployment{
```

Comment:
This is copied into 2 places now, please make a `NewCephFSDeployment()` function, and one for RBD too.